package.json

```diff
@@ -8,10 +8,12 @@
     "react-scripts": "3.1.0"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "PORT=3001 react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "features": "jest -c jest.config.js",
```

Comment:
why did you add this script?

Reply:
We had it in the cooper challange and thought it was relevent here as well. Its job of what i understan is to make it simple to use mocks outside of the test scope. Maybe thats for later or we don't need it at all, I'll remove it!